Extensions/retags.js

```diff
@@ -1,6 +1,6 @@
 //* TITLE       Retags **//
 //* DEVELOPER   new-xkit **//
-//* VERSION     1.0.7 **//
```

Comment:
oh duh my b. I meant to say minor

On Tue, Jul 5, 2016 at 9:45 PM BlackjackKent <notifications@github.com>
wrote:

> In Extensions/retags.js
> <https://github.com/new-xkit/XKit/pull/1006#discussion_r69665513>:
>
> > @@ -1,6 +1,6 @@
> >  //* TITLE       Retags **//
> >  //* DEVELOPER   new-xkit **//
> > -//* VERSION     1.0.7 **//
>
> @nightpool <https://github.com/nightpool> 1.0.7 to 1.0.8 is a patch bump,
> no?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/new-xkit/XKit/pull/1006/files/0f7feeaf08dbdb0e9fc257af0bc491725dc1b632#r69665513>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAORV9HUgip9ko_yCUnJdygQihFaChi6ks5qSwjMgaJpZM4H6Zuu>
> .
>